core/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/AppRoutesTestJsonCodec.scala

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.dsde.workbench.leonardo.http
+
+import java.net.URL
+
+import io.circe.{Decoder, Encoder, KeyDecoder}
+import org.broadinstitute.dsde.workbench.google2.KubernetesSerializableName.ServiceName
+import org.broadinstitute.dsde.workbench.leonardo.JsonCodec._
+
+object AppRoutesTestJsonCodec {
+
+  implicit val createAppEncoder: Encoder[CreateAppRequest] = Encoder.forProduct5(
```

Comment:
These may be only used here in tests, but we could also move these encoder/decoders to core instead

Reply:
in app, I don't think we'll ever need to encode a `CreateAppRequest`....having it in test is good.